roles/openshift_inventory/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- no_log: True  # keep task descriptions legible
+  block:
+
+  - name: Standardize on latest variable names
+    set_fact:
+      # goal is to deprecate deployment_type in favor of openshift_deployment_type.
+      # both will be accepted for now, but code should refer to the new name.
+      # TODO: once this is well-documented, add deprecation notice if using old name.
+      deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+      openshift_deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
```

Comment:
Yeah, alright, that's fine with me.

I don't want to unconditionally set `openshift_release` though due to logic in other roles that checks whether it's defined.